domain/secret/state/state.go

```diff
@@ -468,7 +473,7 @@ func (st State) updateSecret(
 	// the update statement needed.
 	existingSecretQuery := `
 WITH rev AS (
-    SELECT MAX(revision) AS latest_revision
+    SELECT MAX(revision) AS latest_revision, uuid AS latest_revision_uuid
```

Comment:
Given we have a card to remove this WITH clause, can we do that here since we are changing it anyway

Reply:
I would remove all these WITH statements in one PR.